bciers/apps/reporting/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Install dependencies only when needed
+FROM docker.io/node:lts-alpine as deps
```

Comment:
It might be good to specify the exact version of the image, otherwise we have a production build where we don't control the base image

Reply:
Good flag, I've replaced the `FROM`s with a set version (and switched from alpine to debian for consistency with the rest of the project) in eacb7a1c1cd01b35930228a2b93d9528218a7f17.